daffodil-runtime1/src/main/scala/org/apache/daffodil/dpath/XSHexBinary.scala

```diff
@@ -90,14 +90,6 @@ case class XSHexBinary(recipe: CompiledDPath, argType: NodeInfo.Kind)
   val name = "XSHexBinary"
```

Comment:
XSHexBinary mixes in HexBinaryKind, which I think isn't needed anymore? Can that be removed?

+1